### DISCUSSION TITLE
vector tile sources

### DISCUSSION THREAD

**Discussion by AbdulsalamAmin** [pelicanmapping/osgearth#2528] [Support Q&A]
Hi, I'm a GIS specialist working on osgearth. I want to create a vector tile data from sources (eg geoserver in pbf format) with zooming and optimized function for osgearth, for this purpose I read and tested the sources and documents but did not get good results. An example is mbtiles which does not support zoom. I want to make a vector tile like osm, but I don't know exactly how to do it. Thanks to the osgearth team.

**Comment by gwaldron** [comment]
osgEarth has some support for the Mapbox Vector Tiles specification (MVT) so you could research that.
https://github.com/mapbox/mapnik-vector-tile

**Comment by jasonbeverage** [comment]
[tippecanoe](https://github.com/mapbox/tippecanoe) is a great tool for making vector tiles, I'd recommend giving it a shot.
